CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(naoqi_driver)
 
+add_definitions(-std=gnu++11)
```

Comment:
libqi itself requires gnu++11 indeed for reasons related to it's introspection features (if I remember correctly).
The code using libqi should not require it, again, if I remember correctly. I might be wrong but I don't think it's necessary. The extensions are used only in implementation of libqi, not in headers.

BTW The officially recommanded way to add definitions is through the target_compile_definitions() function not through add_definitions().